### DISCUSSION TITLE
Add "watch" mode to CLI options

### DISCUSSION THREAD

**Discussion by bmchristensen** [CleanCut/green#213] [Enhancements]
Some test runners, Jest for example, have CLI options to watch project files and run the tests on the detection of a change to those files. This improves workflow during strict TDD.

Although I may be mistaken, I haven't seen any setting or option to do this with green. I certainly think this would be a useful enhancement.

In a similar vein to Jest, from the CLI:
`green --watch`
`green --watchAll`

**Comment by CleanCut** [comment]
> I haven't seen any setting or option to do this with green.

Correct.  Green does not have this functionality.  I wonder if there is some other utility that one could use to do this sort of thing.  I would imagine there must be something out there where you could just do...

```
watch_stuff myproj 'test_*.py' -c "green ..."
```
